spec/controllers/user/omniauth_callbacks_controller_spec.rb

```diff
@@ -5,24 +5,34 @@
 
   with_tenant(:instance) do
     before { controller.request.env['devise.mapping'] = Devise.mappings[:user] }
+    self::UID = SecureRandom.random_number(2**48)
 
     describe '#facebook' do
-      let(:facebook_data) { OmniAuth.config.mock_auth[:facebook].dup }
+      let(:uid) { SecureRandom.random_number(2**48) }
+      let(:facebook_data) do
+        data = generate(:omniauth_facebook)
```

Comment:
then replace the sequence with a normal factory. factories do not need to generate ActiveRecord objects, the AuthHash should be able to be created in a factory.